REQUIRE

```diff
@@ -4,7 +4,7 @@ Distributions
 Docile
 DualNumbers
 FITSIO
-ForwardDiff 0.0.3 0.1
+ForwardDiff # Currently only required for testing.
```

Comment:
Unfortunately, ```ForwardDiff``` types allocate a ton of memory.  We can't use them as a drop-in replacement for our own derivatives or Hessians.  It might be good to point this out to Miles, though.  He might have some ideas.

```
# ForwardDiff types:
julia> @time for n=1:10000
         foo = sum([ mp_fd.vp[1][i] * mp_fd.vp[1][i] for i=1:length(ids)] )
       end
  0.306251 seconds (2.26 M allocations: 592.808 MB, 23.21% gc time)

# Ordinary types:
julia> @time for n=1:10000
         bar = sum([ mp.vp[1][i] * mp.vp[1][i] for i=1:length(ids)] )
       end
  0.093861 seconds (1.29 M allocations: 22.739 MB, 4.02% gc time)
```